Userland/Libraries/LibWeb/URL/URLSearchParams.cpp

```diff
@@ -200,18 +214,18 @@ DeprecatedString URLSearchParams::get(DeprecatedString const& name)
 }
 
 // https://url.spec.whatwg.org/#dom-urlsearchparams-getall
-Vector<DeprecatedString> URLSearchParams::get_all(DeprecatedString const& name)
+Vector<String> URLSearchParams::get_all(String const& name)
```

Comment:
Could update this to use fallible vector append as well while we're here :)

Reply:
Done:)